CPAC/utils/utils.py

```diff
@@ -287,190 +294,200 @@ def get_zscore(map_node=False, wf_name='z_score'):
-    import nipype.interfaces.utility as util
-    import nipype.interfaces.fsl as fsl
 
     wflow = pe.Workflow(name=wf_name)
 
-    inputNode = pe.Node(util.IdentityInterface(fields=['correlation_file',
-                                                       'timeseries_one_d']),
-                        name='inputspec')
+    inputNode = pe.Node(
+        util.IdentityInterface(fields=["correlation_file", "timeseries_one_d"]),
+        name="inputspec",
+    )
 
     outputNode = pe.Node(
-        util.IdentityInterface(fields=['fisher_z_score_img']),
-        name='outputspec')
+        util.IdentityInterface(fields=["fisher_z_score_img"]), name="outputspec"
+    )
 
     if map_node:
         # node to separate out
         fisher_z_score = pe.MapNode(
-            util.Function(input_names=['correlation_file',
-                                       'timeseries_one_d',
-                                       'input_name'],
-                          output_names=['out_file'],
-                          function=compute_fisher_z_score),
-            name='fisher_z_score',
-            iterfield=['correlation_file'])
+            util.Function(
+                input_names=["correlation_file", "timeseries_one_d", "input_name"],
+                output_names=["out_file"],
+                function=compute_fisher_z_score,
+            ),
+            name="fisher_z_score",
+            iterfield=["correlation_file"],
+        )
     else:
         fisher_z_score = pe.Node(
-            util.Function(input_names=['correlation_file',
-                                       'timeseries_one_d',
-                                       'input_name'],
-                          output_names=['out_file'],
-                          function=compute_fisher_z_score),
-            name='fisher_z_score')
+            util.Function(
+                input_names=["correlation_file", "timeseries_one_d", "input_name"],
+                output_names=["out_file"],
+                function=compute_fisher_z_score,
+            ),
+            name="fisher_z_score",
+        )
 
     fisher_z_score.inputs.input_name = input_name
 
-    wflow.connect(inputNode, 'correlation_file',
-                  fisher_z_score, 'correlation_file')
-    wflow.connect(inputNode, 'timeseries_one_d',
-                  fisher_z_score, 'timeseries_one_d')
-    wflow.connect(fisher_z_score, 'out_file',
-                  outputNode, 'fisher_z_score_img')
+    wflow.connect(inputNode, "correlation_file", fisher_z_score, "correlation_file")
+    wflow.connect(inputNode, "timeseries_one_d", fisher_z_score, "timeseries_one_d")
+    wflow.connect(fisher_z_score, "out_file", outputNode, "fisher_z_score_img")
 
     return wflow
 
 
 def compute_fisher_z_score(correlation_file, timeseries_one_d, input_name):
-    """
-    Computes the fisher z transform of the input correlation map
+    """Compute the fisher z transform of the input correlation map.
```

Comment:
We've got two definitions for `compute_fisher_z_score` (this and https://github.com/FCP-INDI/C-PAC/blob/2cc59aa0752f690988bb67ff671590bcbc0b5fa5/CPAC/sca/utils.py#L22-L91). That one isn't called anywhere; this one doesn't use the timeseries file at all. 
<img width="951" alt="diff" src="https://github.com/FCP-INDI/C-PAC/assets/5974438/92a0e646-76f6-4ab7-9c0b-f3d814cf43c7">